video_processing/modules/gemma3.py

```diff
@@ -56,12 +56,22 @@ def recover_json_from_output(output: str):
 
 @torch.no_grad()
 @torch.inference_mode()
-def run_vlm(model, processor, messages):
+def run_vlm(model, processor, messages_list):
     inputs = processor.apply_chat_template(
-        messages, add_generation_prompt=True, tokenize=True, return_dict=True, return_tensors="pt"
+        messages_list,
+        tokenize=True,
+        return_dict=True,
+        return_tensors="pt",
+        add_generation_prompt=True,
+        padding=True,
+        padding_side="left",
```

Comment:
These two seem to be the enabling params that made it possible. Mind adding a one-line comment on their importance?

Reply:
Done!